ocaml/doc/OMakefile

```diff
@@ -23,7 +23,20 @@ sdk-install: doc-html
 	cp *.js *.html *.css *.png $(SDK)/docs/html/xenserver/
 	cp xenserver/* $(SDK)/docs/html/xenserver/
 
-noarch-install: sdk-install
+.PHONY: doc-html-install
+doc-html-install: doc-html
+	mkdir -p $(DESTDIR)$(DOCDIR)/html/xcp
+	sh -c 'cp -r -L api $(DESTDIR)$(DOCDIR)/html/xcp/'
```

Comment:
I think the reason behind calling `cp` or other commands via `sh` is that some options do not work in a Makefile the same way as in the shell. Having said that, in this particular case we probably don't need it because we don't create any symlinks for the documentation files; in fact on master I have removed the `sh`. On this branch, because I was backporting and wanted to keep the changes to the minimum necessary, I just copied the lines from the original file (to which I presume they had been copied from somewhere else, because I didn't see a special justification in the comment). But I think it's safe to remove it.